_posts/2016-06-16-cold-vs-hot-observables.md

```diff
@@ -383,5 +383,11 @@ this.contacts = http.get('contacts.json')
 {% endraw %}
 {% endhighlight %}
 
+**Note: When using multiple async pipes on streams with default values, the `.share()` operator might cause problems:**
+
+The `share()`will publish the first value of the stream on the first subscription. The first async pipe will trigger that subscription and get that initial value. The second async pipe however will subscribe after that value has already been emitted and therefore miss that value.
```

Comment:
>I think it always causes problems with streams that have initial values

Yes, basically all streams (where at least the first value) comes synchronously. But it would not be a problem with a regular HTTP request for instance. I think it's worth pointing that out to the reader.

>We only want to keep track of the last value so it should be `shareReplay(1)`

Sounds good!